conf/defaults.ini

```diff
@@ -2027,6 +2027,7 @@ enable =
 # This is EXPERIMENTAL. Please, do not use this section
 enable_api = true
 provider = static
+url =
```

Comment:
where is this read? this hole section is missing from sample.ini

Reply:
Totally forgot to update `sample.ini`. Will do. 
It's read in https://github.com/grafana/grafana/blob/main/pkg/setting/setting_openfeature.go#L29

Reply:
@torkelo after thinking a bit more, should it be added to sample.ini given this configuration should not be changed by the users since open feature is in experimental state?

Reply:
maybe not if that is the case